metriq_gym/circuits.py

```diff
@@ -3,6 +3,16 @@
 import math
 import random
 from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter, ParameterVector
+from helpers.lr_qaoa_helpers import PTC_pairs
+import networkx as nx
+from typing import Literal, List, Tuple
+
+GraphType = Literal[
+    "1D", "NL", "FC"
+]  # 1D chain of qubits (1D), Native Layout graph (NL), or Fully connected graph (FC).
+
+EncodingType = Literal["Direct", "PTC"]
```

Comment:
These are already defined in metriq_gym/benchmarks/lr_qaoa.py

Reply:
I import them now from circuits to lr_qaoa